src/UserService.Business/Commands/Certificate/EditCertificateCommand.cs

```diff
@@ -23,111 +25,113 @@
 
 namespace LT.DigitalOffice.UserService.Business.Commands.Certificate
 {
-    public class EditCertificateCommand : IEditCertificateCommand
+  public class EditCertificateCommand : IEditCertificateCommand
+  {
+    private readonly IAccessValidator _accessValidator;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IUserRepository _userRepository;
+    private readonly ICertificateRepository _certificateRepository;
+    private readonly IPatchDbUserCertificateMapper _mapper;
+    private readonly ICreateImageDataMapper _createImageDataMapper;
+    private readonly IRequestClient<ICreateImagesRequest> _rcImage;
+    private readonly ILogger<EditCertificateCommand> _logger;
+
+    private Guid? GetImageId(AddImageRequest addImageRequest, List<string> errors)
```

Comment:
make method async

Reply:
certs will be done in https://github.com/LT-Students/DigitalOffice-UserService/pull/122